docs/troubleshooting-guide/cant-log-in.md

```diff
@@ -0,0 +1,49 @@
+# People can't log in to Metabase
+
+You are supposed to be able to log in to Metabase, but:
+
+- you can't see the login page, or
+- your credentials aren't accepted.
+
+## Are you using the right URL?
+
+- Do you have the port as well as the host?
+- Has someone moved the Metabase instance (e.g., it was in dev and now it's in production)?
+
+## Are the credentials stored by Metabase or is the account managed using Single Sign-On?
+
+- Other steps depend on knowing this
+- FIXME: can people look at their own profile to find out how their credentials are managed?
```

Comment:
The tell here should be obvious when they sign in. They should have an option to use the third party identity provider, or "sign in with email".